src/sidebar/util/annotation-metadata.js

```diff
@@ -306,3 +306,37 @@ export function quote(annotation) {
   const quoteSel = target.selector.find(s => s.type === 'TextQuoteSelector');
   return quoteSel ? /** @type {TextQuoteSelector}*/ (quoteSel).exact : null;
 }
+
+/**
+ * Has this annotation been edited subsequent to its creation?
+ *
+ * @param {Annotation} annotation
+ * @return {boolean}
+ */
+export function hasBeenEdited(annotation) {
+  let hasBeenEdited = false;
+  if (!annotation.updated) {
+    return hasBeenEdited;
+  }
+  // updated and created are strings
+  const maybeEdited =
+    annotation.updated && annotation.created !== annotation.updated;
+  if (maybeEdited) {
+    try {
```

Comment:
`created` and `updated` "should" be parse-able ISO 8601 date strings, but I am loathe to make any assumptions.